Modules/Project/Services/ProjectService.php

```diff
@@ -379,4 +382,31 @@ public function getMailDetailsForZeroExpectedHours()
 
         return $projectDetails;
     }
+
+    public function projectFTEExport($filters, $request)
+    {
+        $employees = Employee::query()->applyFilters($filters)
+            ->get();
+
+        $employees = $this->formatProjectFTEForExportAll($employees);
+        $currentTimeStamp = Carbon::now();
+
+        return Excel::download(new ProjectFTEExport($employees), "FTE-$currentTimeStamp->year$currentTimeStamp->month$currentTimeStamp->day.xlsx");
+    }
+
+    private function formatProjectFTEForExportAll($employees)
+    {
+        $teamMembers = [];
+        foreach ($employees as $employee) {
+            if ($employee->user) {
+                foreach ($employee->user->activeProjectTeamMembers as $activeProjectTeamMember) {
+                    $teamMember = [];
+                    array_push($teamMember, $employee->name, number_format($employee->user->ftes['main'], 2), $activeProjectTeamMember->project->name, number_format($activeProjectTeamMember->fte, 2));
+                    array_push($teamMembers, $teamMember);
+                }
+            }
+        }
```

Comment:
Try to follow the negation first approach which makes code more readable. 

```suggestion
        foreach ($employees as $employee) {
             if (!$employee->user) {
             continue;
             }

          foreach ($employee->user->activeProjectTeamMembers as $activeProjectTeamMember) {
                         $teamMember = [
                          $employee->name, 
                          number_format($employee->user->ftes['main'], 2), 
                          $activeProjectTeamMember->project->name, 
                          number_format($activeProjectTeamMember->fte, 2)
                      ];
                    $teamMembers[] = $teamMember
                }
        }
```